lambdas/service/app.py

```diff
@@ -677,8 +681,8 @@ def validate_organism_age_filter(values):
             raise BRE(e)
 
 
-def validate_field(field: str):
-    if field not in app.metadata_plugin.field_mapping:
+def validate_field(field: str, *extras):
+    if field not in app.metadata_plugin.field_mapping and field not in extras:
```

Comment:
```suggestion
    if field not in app.concrete_fields and field not in app.synthetic_fields:
```

Reply:
Note that this function is also used to validate the `sort` parameter, where `accessible` is _not_ supported, so the caller needs to be able to control whether to include the synthetic fields or not.